src/scss/_elements.text.scss

```diff
@@ -0,0 +1,12 @@
+a[href]:not([class]) {
+  color: $blue;
+  transition: all ease-in-out 150ms;
```

Comment:
TIL that transition effects don't work on `text-decoration` 💡Looks like we could replace that with a `border-bottom` so that this transition works.